strapi/strapi
# Chore/iteratively implementation

### What does it do?

This PR introduces Iteratively to Strapi. Iteratively is a tool that allows for easier tracking plan creation and tracking validation. This PR adds generated Itly library to the core Strapi package and funnels all `strapi.telemetry.send()` calls to go through Iteratively validation process. After validation all event data is forwarded to Strapi analytics tracker as before.

### Why is it needed?

It is part of the Iteratively implementation initiative.

### How to test it?

Run the "getstarted" project for testing purposes. You can change analytics tracker url in the `/packages/core/strapi/lib/services/metrics/destination.js` file to point tracking output to a localhost server to see what results you get. 

### Related issue(s)/PR(s)

N/A
